src/main/java/seedu/moneygowhere/commands/ConsoleCommandConvertCurrency.java

```diff
@@ -0,0 +1,35 @@
+package seedu.moneygowhere.commands;
+
+import seedu.moneygowhere.data.currency.CurrencyManager;
+import seedu.moneygowhere.data.expense.Expense;
+
+import java.math.BigDecimal;
+
+/**
+ * Stores the Convert-Currency command and its arguments.
+ */
+@SuppressWarnings("FieldMayBeFinal")
+public class ConsoleCommandConvertCurrency extends ConsoleCommand {
+    private int expenseIndex;
+    private String currency;
+
+    public ConsoleCommandConvertCurrency(int expenseIndex, String currency) {
+        this.expenseIndex = expenseIndex;
+        this.currency = currency;
+    }
+
+    public int getExpenseIndex() {
+        return expenseIndex;
+    }
+
+    public String getCurrency() {
+        return currency;
+    }
+
+    public void changeCurrency(Expense expense, CurrencyManager currencyManager) {
+        Float newAmount = currencyManager.exchangeCurrency(expense, currency);
```

Comment:
Do use BigDecimal instead of floats or doubles when representing currency amounts as floating point arithmetic will cause losses in precision.